exchange/non_bid_reason.go

```diff
@@ -13,6 +24,9 @@ const (
 	ResponseRejectedBelowDealFloor         NonBidReason = 304 // Response Rejected - Bid was Below Deal Floor
 	ResponseRejectedCreativeSizeNotAllowed NonBidReason = 351 // Response Rejected - Invalid Creative (Size Not Allowed)
 	ResponseRejectedCreativeNotSecure      NonBidReason = 352 // Response Rejected - Invalid Creative (Not Secure)
+	ErrorTimeout                           NonBidReason = 101 // Error - Timeout
+	ErrorGeneral                           NonBidReason = 100
+	ErrorBidderUnreachable                 NonBidReason = 103 // Error - Bidder Unreachable
```

Comment:
Use `openrtb3.NoBidReason `

Reply:
Why do you think it's better to use `openrtb3.NoBidReason`? That's a different enumeration. It's most appropriate to use the existing `NonBidReason`. Please revert this change.